### PR TITLE
Create a simple processor for RLIMS-P results.

### PR DESCRIPTION
**Pull request by pagreene**
Create a processor for RLIMS-P json, and an API to use their webservice.

### CODE REVIEW

**Comment by pagreene** [comment]
Travis tests are failing for unrelated reasons.

File: indra/tests/test_rlimsp.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    rp = rlimsp.process_from_webservice('PMC3717945', with_grounding=False)
+    assert len(rp.statements) == 33, len(rp.statements)
+
+
+def test_grounded_endpoint_with_pmids():
+    pmid_list = ['16403219', '22258404', '16961925', '22096607']
+    stmts = []
+    for pmid in pmid_list:
+        rp = rlimsp.process_from_webservice(pmid, id_type='pmid',
+                                            with_grounding=False)
```
Comment (johnbachman) [inline indra/tests/test_rlimsp.py]: Looks like this test is for the grounded endpoint but uses `with_grounding=False`

Comment (pagreene) [reply indra/tests/test_rlimsp.py]: Ah yes. The name did not change to keep up with reality.

Comment (johnbachman) [reply indra/tests/test_rlimsp.py]: Then probably also worth adding at least one test of the grounded endpoint as well

Comment (pagreene) [reply indra/tests/test_rlimsp.py]: The first test is of the grounded endpoint. Do you mean a grounded endpoint with pmids?

Comment (pagreene) [reply indra/tests/test_rlimsp.py]: I added some more tests.

Comment (johnbachman) [reply indra/tests/test_rlimsp.py]: Got it--didn't notice that with_grounding=True is the default in the first test

**Pull request merged**